<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PositionDialog</class>
 <widget class="QDialog" name="PositionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>291</width>
    <height>180</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>291</width>
    <height>180</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>291</width>
    <height>180</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Change Coordinates</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QPushButton" name="centerButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>70</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Center</string>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>251</width>
     <height>25</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Close</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>10</y>
     <width>121</width>
     <height>56</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="1" column="1">
     <widget class="QDoubleSpinBox" name="ySpinBox"/>
    </item>
    <item row="0" column="1">
     <widget class="QDoubleSpinBox" name="xSpinBox"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>X</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Y</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>141</width>
     <height>141</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QPushButton" name="leftButton">
      <property name="minimumSize">
       <size>
        <width>41</width>
        <height>41</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>41</width>
        <height>41</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="rightButton">
      <property name="minimumSize">
       <size>
        <width>41</width>
        <height>41</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>41</width>
        <height>41</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="upButton">
      <property name="minimumSize">
       <size>
        <width>41</width>
        <height>41</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>41</width>
        <height>41</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="downButton">
      <property name="minimumSize">
       <size>
        <width>41</width>
        <height>41</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>41</width>
        <height>41</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PositionDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>210</x>
     <y>264</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PositionDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>210</x>
     <y>264</y>
    </hint>
    <hint type="destinationlabel">
     <x>274</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
